src/PayID/pay-id-utils.ts

```diff
@@ -1,39 +1,37 @@
 // Disable multiple classes to accommodate the switch to idiomatic style naming.
-// TODO(keefertaylor): Remove this when migration is complete.
-/* eslint-disable  max-classes-per-file */
-
+/* eslint-disable import/no-deprecated */
 import PayIdComponents, { PayIDComponents } from './pay-id-components'
 
 /**
  * A static utility class for Pay ID functionality.
  */
-export default class PayIdUtils {
+const payIdUtils = {
   /**
    * Parse a PayID string to a set of PayIDComponents object
    *
-   * @param parsePayID - The input Pay ID.
+   * @param payID - The input Pay ID.
    * @returns A PayIDComponents object if the input was valid, otherwise undefined.
    */
-  public static parsePayID(payID: string): PayIdComponents | undefined {
-    if (!PayIdUtils.isASCII(payID)) {
-      return
+  parsePayID(payID: string): PayIdComponents | undefined {
+    if (!this.isASCII(payID)) {
+      return undefined
```

Comment:
The [consistent-return](https://eslint.org/docs/rules/consistent-return) rule requires being explicit with your `return` statements if your function contains a `return` statement that returns something other than undefined (like this function with `return new PayIdComponents()`.

Reply:
woo. I actually prefer this.